hmda/src/main/scala/hmda/validation/rules/lar/validity/V632_3.scala

```diff
@@ -23,7 +23,9 @@ object V632_3 extends EditCheck[LoanApplicationRegister] {
     val coEthnicity = lar.coApplicant.ethnicity
     when(coEthnicity.ethnicityObserved is equalTo(NotVisualOrSurnameEthnicity)) {
       (coEthnicity.ethnicity1 is containedIn(validEthnicityList)) or
-        (coEthnicity.otherHispanicOrLatino not empty)
+        ((coEthnicity.ethnicity1 is equalTo(EmptyEthnicityValue)) and
```

Comment:
Visually, this code is a bit confusing, I believe that it is `(c1 or c2) and c3` but could you put parenthesis in to confirm?